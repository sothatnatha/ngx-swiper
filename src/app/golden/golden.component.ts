import { Component, OnInit } from '@angular/core';

@Component({
  selector: 'app-golden',
  templateUrl: './golden.component.html',
  styleUrls: ['./golden.component.scss']
})
export class GoldenComponent implements OnInit {

  constructor() { }

  ngOnInit(): void {
  }

}
