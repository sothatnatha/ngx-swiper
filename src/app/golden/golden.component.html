<app-home></app-home>
<div class="golden">
    <img src="../../assets/golden-retriever.png" alt="">
</div>
